surveys/notes/README.md

```diff
@@ -0,0 +1,3 @@
+# Notes
```

Comment:
Should this whole section be called `docs` or `process` instead?

Reply:
I thought we might make notes on other things too, so have kept process to one file. I thought 'notes' better reflects the quality of the documentation than 'docs' :-)